src/components/ContainerHomeLogs.react.js

```diff
@@ -40,17 +40,20 @@ let prevBottom = 0;
 module.exports = React.createClass({
   getInitialState: function(){
     return {
-      fontSize: 10
+      fontSize: 10,
+      follow: true,
     };
   },
   onFontChange: function(event){
-    this.setState({
-      fontSize: event.target.value
-    });
+    let $target = event.target;
+    this.setState((prevState)=>({
+      fontSize: $target.value,
+      follow: prevState.follow
+    }));
   },
   componentDidUpdate: function () {
     var node = $('.logs').get()[0];
-    node.scrollTop = node.scrollHeight;
+    node.scrollTop = (this.state.follow)? node.scrollHeight: 0;
```

Comment:
As soon as I "unfollow" I end up being thrown to the top of the logs - Anyway to capture the current scroll position and simply keep the user there?

Reply:
Thanks, hadn't thought of this before. Going to fix this ASAP